src/js/media-store/state-mediator.ts

```diff
@@ -428,14 +448,25 @@ export const stateMediator: StateMediator = {
         if (noVolumePref) return;
         /** @TODO How should we handle globalThis dependencies/"state ownership"? (CJP) */
         try {
+          const { media } = stateOwners;
+          if (!media) return;
+
           const volumePref = globalThis.localStorage.getItem(
             'media-chrome-pref-volume'
           );
+          const isMuted =
+            globalThis.localStorage.getItem('is-muted') === 'true';
+
+          media.muted = isMuted;
+          if (!isMuted && volumePref !== null) {
+            media.volume = +volumePref;
+          }
```

Comment:
I think this shouldn't be here, the muted localStorage logic should be part of the the `mediaMuted` property https://github.com/muxinc/media-chrome/pull/1088/files#diff-333677ad6c1f12477baa3069c5713d46dfcfe6d53826b6743860ca2d7eecb49cR383

it'll require a `stateOwnersUpdateHandlers` as well.
basically you need to clone how mediaVolume does it but then for mediaMuted. no need to mix the 2 I think.